pkg/kubelet/apis/config/validation/validation.go

```diff
@@ -203,6 +203,15 @@ func ValidateKubeletConfiguration(kc *kubeletconfig.KubeletConfiguration, featur
 		allErrors = append(allErrors, fmt.Errorf("invalid configuration: memorySwap.swapBehavior cannot be set when NodeSwap feature flag is disabled"))
 	}
 
+	if localFeatureGate.Enabled(features.KubeletCrashLoopBackOffMax) {
+		switch {
+		case kc.CrashLoopBackOff == nil:
+		case kc.CrashLoopBackOff.MaximumBackOffPeriod == nil:
```

Comment:
If we have a default, then you can just validate this as being not-nil and within range.